src/atoms/CarouselSlot.js

```diff
@@ -11,8 +11,10 @@ export const CarouselSlot = styled(Col)`
 
 		return css`
 			margin-right: ${marginRight};
-			margin-bottom: 0;
 			padding: 0;
+			article {
```

Comment:
How about we make this a general thing targeting whatever contents might populate the CarouselSlot?

```suggestion

			> * {
```